src/main/java/com/dawn/jat/illuminati/post/controller/PostController.java

```diff
@@ -28,12 +28,14 @@
      */
     @GetMapping(value = "/{slug}")
     public ResponseEntity<Object> getPostBySlug(@PathVariable("slug") String slug) {
-        Optional<PostEntity> postEntity = postService.findBySlug(slug);
+        List<PostEntity> postEntity = postService.findBySlug(slug);
```

Comment:
Why do we switch from Optional to List? 
This is supposed to return just 1 item?

Reply:
can we try to change `SuccessResponse` data into Object. If we change this everytime there is a new kind of data, it's not good
![image](https://user-images.githubusercontent.com/7911873/74700660-d7c5d680-5236-11ea-8412-74160d72d8fd.png)

Reply:
Updated